samples/batch/python/python-client/main.py

```diff
@@ -143,25 +143,32 @@ def transcribe():
     # Specify transcription properties by passing a dict to the properties parameter. See
     # https://docs.microsoft.com/azure/cognitive-services/speech-service/batch-transcription#configuration-properties
     # for supported parameters.
-    properties = {
-        # "punctuationMode": "DictatedAndAutomatic",
-        # "profanityFilterMode": "Masked",
-        # "wordLevelTimestampsEnabled": True,
-        # "diarizationEnabled": True,
-        # "destinationContainerUrl": "<SAS Uri with at least write (w) permissions for an Azure Storage blob container that results should be written to>",
-        # "timeToLive": "PT1H"
-    }
+    properties = cris_client.TranscriptionProperties()
```

Comment:
Can we take this opportunity to move away from "cris"?

Reply:
done